### ISSUE TITLE
review grade band widths (eg number AU grades in expert might be too high)

### ISSUE THREAD

**Issue by scd** [theCrag/website#1927]
From Will M....

Hi guys
just wondering why the "expert" (red) grade band contains 8 grades (25-32), when the intermediate (yellow, grades 13-18) and experienced (orange, grades 18-24) bands contain 6 grades?

Surely each band should contain an equal number of grades? (At least, once you're above the 12 green beginner grades). I suggest that expert should be 25-30, then elite is 31-36, then you start a new band (under whatever suitable name&colour) at grade 37 which will see you through until someone does a 43 (!).  

cheers

**Comment by scd** [comment]
This was originally done with some sort of statistical analysis, looking at ascent numbers. The ascent profile distinctly changes at certain grades. This will be worth revisiting at some stage as we did not have a large number of ascents at the higher grades.

**Comment by willmonks** [comment]
Thanks Simon, couldn't remember my github login the other day. Hardly any 32+ climbers are using the site so any "distinct cutoff" in the 30+ ascent numbers probably has no statistical certainty. 8a.nu has tonnes more top-end users' data which, although cumbersome to review, doesn't appear to suggest any distinct cutoff to justify the wider "Expert" band. Logic/stats would suggest that hard ascents will always tail off like a bell curve, so the bands should just be of constant size IMO.

**Comment by scd** [comment]
I presume you are making the comments in regards to AU grades. Other grading systems would need to be considered.

FYI there are a couple of additional things I would like to sort out before tackling this. In particular, the system assumes that grades do not straddle two bands (ie have two colors) for any grade system. I don't like this assumption because it forced me to make some compromises and is very difficult to change the bands while this assumption in in play.

**Comment by willmonks** [comment]
yes Ewbank grades in my post above.  AU, US & French grades (plus maybe others) are in lockstep above grade 25/7b/5.12b, so at least in the expert/elite bands it's all 1:1 between the dominant grading systems.

**Comment by brendanheywood** [comment]
Referenced from this discussion:

https://www.thecrag.com/discussion/1412501241/can-you-add-a-new-colour-for-the-difficulty-of-this-climb--#m1412954817

**Comment by scd** [comment]
There seems to be appetite for this. The bands are currently called:

 * Beginner
 * Intermediate
 * Experienced
 * Expert
 * Elite
 * ????

What shall we call the next band?

**Comment by brendanheywood** [comment]
There is two issues here

1) how many bands are there
2) how many grades, and which grades go in each band

2 is 90% of the problem and we can solve it just by realigning the grades. Over time the idea of elite changes, I don't think we need a new band, I think we should just change what grades are in elite.

I still would like to entertain and prototype the idea that we replace the discrete bands with a smooth spectrum of colours

**Comment by scd** [comment]
I don't disagree with anything you have said, but they are way more work and have further dependencies. In particular we have artificially tried to make the transitions across grades for all grading systems to align with the band transitions. We need to remove this unnessary and artificial requirement and get more accurate grade transitions. This is probably a fair bit of work which we would need to complete before we change band widths or make it a continuous spectrum.

This issue will not be resolved for a while. The real trigger here is when people start paying attention to CPR and insisting on more accurate grade conversions.

**Comment by brendanheywood** [comment]
The continuous spectrum idea I see as purely cosmetic workaround and I think a much better direction than introducing a new band. The bands themselves 'elite', 'expert' etc to me have always smelled too much like their own grade system, with all the same warts as other grade systems like the uk 'hard', 'hard severe', 'hard very severe'. The reality is that lots of kids at the gym are climbing 19's pretty quickly, calling this 'Experienced' is increasingly meaningless. The actual grade ranges we still need to refine but that is mostly independent of the visual color scheme we use.

In terms of effort this is quite low, like this is a super rough working prototype. This solves a few problems:

* you go to a crag like coolum where almost everything is red, then the red becomes meaningless, as you can't easily tell the routes apart visually. Using a spectrum you can.
* you have a 24 next to a 25 and the difference seems much greater than it is. The jumps between the grades below is small but enough to actually tell adjacent grades apart.
* we'd still leave the grade bars as is (until we replace them with the new bar charts Gabe is working on)


To get this prod ready we just need to make sure that everywhere we use a grade I have access to a raw internal grade, and spend a bit of time refining the actual gradient colors (this is also on Gabe's todo list). 

![image](https://user-images.githubusercontent.com/187449/29757458-4e26d0ec-8bef-11e7-8490-56e031b0d573.png)
